appveyor.yml

```diff
@@ -18,13 +18,13 @@ environment:
 
   matrix:
 
-    - CONDA_ROOT: "C:\\Miniconda3_64"
+    - CONDA_ROOT: "C:\\Miniconda36_64"
```

Comment:
this doesn't actually do anything other than create a directory for us to *install* into